submissions/DmytroMukhin/HTML_CSS_Hooli-style_Popup/index.css

```diff
@@ -0,0 +1,301 @@
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.navigation__item {
+  margin-right: 1rem;
+}
+
+.navigation__item:last-child {
+  margin-right: 0;
+}
+
+.navigation__link {
+  padding: 1rem;
+  color: #214d54;
+}
+
+.navigation__link:hover,
+.navigation__link:focus {
+  outline: 0.01rem solid #54bab9;
+}
+
+.popup {
+  position: relative;
+}
+
+.popup__input,
+.popup__input--more {
+  position: absolute;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+}
+
+.popup__label {
+  display: block;
+  padding: 1.575rem;
+  background-image: url(icons/popup-button.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.popup__input:hover + .popup__label,
+.popup__input:focus + .popup__label {
+  cursor: pointer;
+  outline: 0.01rem solid #54bab9;
+}
+
+.popup__list {
+  position: absolute;
+  top: 4rem;
+  left: -15rem;
+  width: 22rem;
+  max-height: 25rem;
+  display: flex;
+  justify-content: space-between;
+  padding: 1.5rem 1.5rem 0.5rem;
+  flex-wrap: wrap;
+  list-style: none;
+  border-radius: 0.5rem;
+  overflow-x: hidden;
+  overflow-y: auto;
+  box-shadow: rgba(0, 0, 0, 0.2) 0px 0.4rem 0.8rem;
+  scrollbar-gutter: stable;
+  background-color: #f6f6f6;
+  display: none;
+}
+
+.popup__item {
+  width: 30%;
+  margin-bottom: 0.65rem;
+}
+
+.popup__more {
+  width: 100%;
+  text-align: center;
+  font-size: 1.2rem;
+  letter-spacing: 0.2rem;
+}
+
+.popup__label--less {
+  display: none;
+}
+
+.popup__label--more,
+.popup__label--less {
+  cursor: pointer;
+  color: #214d54;
+}
+
+.popup__input--more:hover + .popup__label--more,
+.popup__input--more:focus + .popup__label--more,
+.popup__input--more:hover ~ .popup__label--less,
+.popup__input--more:focus ~ .popup__label--less {
+  border-bottom: 0.1rem solid #54bab9;
+  margin-bottom: -0.1rem;
+}
+
+.popup__input--more:checked + .popup__label--more {
+  display: none;
+}
+
+.popup__input--more:checked ~ .popup__label--less {
+  display: inline-block;
+}
+
+.popup__link {
+  height: 80%;
+  padding: 0.9rem 0;
+}
+
+.popup__link:hover,
+.popup__link:focus {
+  outline: 0.01rem solid #54bab9;
+}
+
+.popup__figure {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  height: 80%;
+}
+
+.popup__img {
+  width: 2.8rem;
+  height: auto;
+}
+
+.popup__figcaption {
+  margin-top: 0.65rem;
+  font-size: 0.8rem;
+  letter-spacing: 0;
+  color: #214d54;
+}
+
+.popup__list--more {
+  margin-top: 0.7rem;
+  justify-content: space-between;
+  padding: 0;
+  flex-wrap: wrap;
+  list-style: none;
+  display: none;
+}
+
+.popup__input:checked ~ .popup__list,
+.popup__input--more:checked ~ .popup__list--more {
+  display: flex;
+}
+
+::-webkit-scrollbar {
+  width: 0.3rem;
+  height: 0.3rem;
+}
+
+::-webkit-scrollbar-button {
+  background-color: #e1eaea;
+}
+
+::-webkit-scrollbar-track-piece {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar
pay attention that it can't be used in production

Reply:
thank you, I fixet